helmfile/charts/sc-logs-retention/scripts/logs-retention.sh

```diff
@@ -35,15 +35,14 @@ declare -a SC_LOG_BACKUPS_REST
 mapfile -t SC_LOG_BACKUPS_LATEST < <(echo "${SC_LOG_BACKUPS}" | tail -n "${RETENTION_DAYS}")
 mapfile -t SC_LOG_BACKUPS_REST <   <(echo "${SC_LOG_BACKUPS}" | head -n "-${RETENTION_DAYS}")
 
+if [ ${#SC_LOG_BACKUPS_REST[*]} -eq 0 ]; then
+echo "No backups were found for automatic removal"; else
 echo "Listing ${RETENTION_DAYS} latest backups"
 echo "${SC_LOG_BACKUPS_LATEST[@]}"
```

Comment:
I would like to have all this properly indented. Like this:

```suggestion
if [ ${#SC_LOG_BACKUPS_REST[*]} -eq 0 ]; then
  echo "No backups were found for automatic removal";
else
  echo "Listing ${RETENTION_DAYS} latest backups"
  echo "${SC_LOG_BACKUPS_LATEST[@]}"
```

And so on...